iroha/tests/test_config.json

```diff
@@ -1,9 +1,22 @@
 {
   "TORII_URL": "127.0.0.1:1338",
   "BLOCK_TIME_MS": 100,
-  "KURA_INIT_MODE": "strict",
+  "KURA_INIT_MODE": "Strict",
   "KURA_BLOCK_STORE_PATH": "./blocks",
-  "IROHA_TRUSTED_PEERS": [{"address":"127.0.0.1:1337", "public_key":"[101, 170, 80, 164, 103, 38, 73, 61, 223, 133, 83, 139, 247, 77, 176, 84, 117, 15, 22, 28, 155, 125, 80, 226, 40, 26, 61, 248, 40, 159, 58, 53]"}, {"address":"localhost:1338", "public_key":"[101, 170, 80, 164, 103, 38, 73, 61, 223, 133, 83, 139, 247, 77, 176, 84, 117, 15, 22, 28, 155, 125, 80, 226, 40, 26, 61, 248, 40, 159, 58, 53]"}, {"address": "195.162.0.1:23", "public_key":"[101, 170, 80, 164, 103, 38, 73, 61, 223, 133, 83, 139, 247, 77, 176, 84, 117, 15, 22, 28, 155, 125, 80, 226, 40, 26, 61, 248, 40, 159, 58, 53]"}],
-  "IROHA_PUBLIC_KEY": "[101, 170, 80, 164, 103, 38, 73, 61, 223, 133, 83, 139, 247, 77, 176, 84, 117, 15, 22, 28, 155, 125, 80, 226, 40, 26, 61, 248, 40, 159, 58, 53]",
-  "IROHA_PRIVATE_KEY": "[113, 107, 241, 108, 182, 178, 31, 12, 5, 183, 243, 184, 83, 0, 238, 122, 77, 86, 20, 245, 144, 31, 128, 92, 166, 251, 245, 106, 167, 188, 20, 8, 101, 170, 80, 164, 103, 38, 73, 61, 223, 133, 83, 139, 247, 77, 176, 84, 117, 15, 22, 28, 155, 125, 80, 226, 40, 26, 61, 248, 40, 159, 58, 53]"
+  "TRUSTED_PEERS": [
+    {
+      "address": "127.0.0.1:1337",
+      "public_key": {"inner": [101, 170, 80, 164, 103, 38, 73, 61, 223, 133, 83, 139, 247, 77, 176, 84, 117, 15, 22, 28, 155, 125,     80, 226, 40, 26, 61, 248, 40, 159, 58, 53]}
+    },
+    {
+      "address": "localhost:1338",
+      "public_key": {"inner": [101, 170, 80, 164, 103, 38, 73, 61, 223, 133, 83, 139, 247, 77, 176, 84, 117,     15, 22, 28, 155, 125, 80, 226, 40, 26, 61, 248, 40, 159, 58, 53]}
+    },
+    {
+      "address": "195.162.0.1:23",
+      "public_key": {"inner": [101, 170, 80, 164, 103, 38, 73, 61, 223, 133, 83, 139, 247, 77, 176, 84, 117, 15, 22, 28, 155, 125, 80, 226, 40, 26, 61, 248, 40, 159, 58, 53]}
+    }
```

Comment:
Let's move trusted peers to a separate file as I think we agreed. Now with `serde_json` it should be simple. Or you want to do it in a separate PR?

Reply:
This and previous comment will be addressed in the PR with `config` decomposition